File: spec/javascript/test_imports_abs_rel.js
var assert = require('assert');
var testHelper = require('testHelper');

testHelper('ImportsAbsRel', 'src/fixed_struct.bin', function(r, ImportsAbsRel) {

  assert.strictEqual(r.one, 80);
  assert.strictEqual(r.two.one, 65);
  assert.strictEqual(r.two.two.one, 67);
});

File: spec/python/test_imports_abs_rel.py
import unittest

from imports_abs_rel import ImportsAbsRel

class TestImportsAbsRel(unittest.TestCase):
    def test_imports_abs_rel(self):
        with ImportsAbsRel.from_file('src/fixed_struct.bin') as r:

            self.assertEqual(r.one, 80)
            self.assertEqual(r.two.one, 65)
            self.assertEqual(r.two.two.one, 67)
